File: Python/Examples/leap_year.py
# Python Program to check given year is leap year.

year = int(input("Enter year: "))
if (year % 4 == 0 and year % 100 != 0 or year % 400 == 0):
    print("The year is leap year!")
else:
    print("The year isn't leap year!")

File: C++/Examples/LeapYear.cpp
#include <iostream>
using namespace std;

int main() {
	int year;
	cout << "Enter an year: ";
	cin >> year;
	if (year % 4 == 0) {
		if (year % 100 == 0) {
			if (year % 400 == 0)
				cout << year << " is leap year." << endl;
			else
				cout << year << " isn't leap year." << endl;
		}
		else
			cout << year << " is a leap year." << endl;
	}
	else
		cout << year << " isn't leap year." << endl;
	return 0;
}
